.circleci/config.yml

```
@@ -78,41 +78,84 @@ jobs:
           name: Run jasmine tests (part C)
           command: ./.circleci/test.sh no-gl-flaky-jasmine
 
-  stable-image:
-    docker:
-      - image: plotly/testbed:latest
+  make-baselines:
     parallelism: 4
-    working_directory: /var/www/streambed/image_server/plotly.js/
+    docker:
+      - image: circleci/python:3.8.9
+    working_directory: ~/plotly.js
     steps:
       - attach_workspace:
-          at: /var/www/streambed/image_server/
+          at: ~/
+      - run:
+          name: which pip3 version
+          command: which pip3 && pip3 --version
       - run:
-          name: Run and setup container
+          name: install kaleido v0.2.1
+          command: python3 -m pip install kaleido==0.2.1
```

```suggestion
          name: install kaleido
          command: pip install kaleido==0.2.1
```
again, in this env we should only have one pip to worry about... also let's not repeat the version number. (applies several places below too)